src/steps/artifact-injection.ts

```diff
@@ -7,6 +7,13 @@ const log = debug('monofo:steps:artifact-injection');
 export const ARTIFACT_INJECTION_STEP_KEY = 'monorepo-inject-artifacts';
 const ARTIFACT_INJECTION_STEP_LABEL = `:crystal_ball:`;
 
+function copyArtifactCommand(artifact: string, buildId: string): string {
+  return (
+    `# Copy ${artifact} from ${buildId} into current build\n` +
+    `buildkite-agent artifact download '${artifact}' . --build '${buildId}' && buildkite-agent artifact upload '${artifact}' &`
```

Comment:
As a possible improvement, we could use the `artifact search` API, and either use a process redirection (`buildkite-agent artifact upload <(curl -sL ${downloadUrl})`) or a named pipe to work around `upload` not taking stdin.